dali/pipeline/operators/reader/coco_reader_op.h

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_READER_COCO_READER_OP_H_
+#define DALI_PIPELINE_OPERATORS_READER_COCO_READER_OP_H_
+
+#include <fstream>
+#include <map>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "dali/pipeline/operators/reader/reader_op.h"
+#include "dali/pipeline/operators/reader/loader/file_loader.h"
+#include "dali/pipeline/operators/reader/parser/coco_parser.h"
+#include "dali/util/json.h"
+
+#define FIND_IN_JSON(im, it, field)   \
+      auto it = im.find("field");     \
+      DALI_ENFORCE(it != im.end(), "`field` not found in JSON annotions file");
+
+#define GET_FORM_JSON(im, field, type)      \
```

Comment:
Is this meant to be `GET_FROM_JSON` instead of `GET_FORM_JSON`?

Reply:
Shame, done.